inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/job/Job.java

```diff
@@ -83,8 +88,21 @@ public void setJobInstanceId(String jobInstanceId) {
      * @return taskList
      */
     public List<Task> createTasks() {
+        return getTasks(this.jobConf);
+    }
+
+    /**
+     * build task from job config
+     *
+     * @param jobConf
```

Comment:
Please add a description for the param and return.

Reply:
done